pom.xml

```diff
@@ -97,6 +97,7 @@
           <preBuildHookScript>setup.bsh</preBuildHookScript>
           <postBuildHookScript>verify.bsh</postBuildHookScript>
           <addTestClassPath>true</addTestClassPath>
+          <streamLogs>true</streamLogs>
```

Comment:
We do not want debugging information printed to the console by default.  
Comment out line 100 and add a comment the enabling this properly will print information to the console.